### ISSUE TITLE
Is it possible to change the "Clear" and "Filter" buttons?

### ISSUE THREAD

**Issue by Xizario** [telerik/kendo-react#1208]
Is it possible to change the "Clear" and "Filter" buttons? 
I want to render my own buttons with icons. I couldn't find anything about it or do I need to make a completely new filter? (preferable don't want to do that)

For some reason the checkbox filter and the normal filter have different way of button presentation.

![image](https://user-images.githubusercontent.com/2089879/151680483-f635e8ce-335a-40b3-9323-ec928ba5e356.png)

![image](https://user-images.githubusercontent.com/2089879/151680498-3eb4f581-0696-4b79-9e6e-a09f62928acd.png)

_Originally posted by @Tommuh in https://github.com/telerik/kendo-react/issues/28#issuecomment-1025004461_

**Comment by kdikov82** [comment]
A temporary workaround:
[https://stackblitz.com/edit/react-rhm6ae?file=app%2FcolumnMenu.jsx,index.html](https://stackblitz.com/edit/react-rhm6ae?file=app%2FcolumnMenu.jsx,index.html)

```
export const ColumnMenuCheckboxFilter = (props) => {
  return (
    <div className="checkboxfilter-wrapper">
      <GridColumnMenuCheckboxFilter
```

```
<style>
      .checkboxfilter-wrapper .k-filter-menu-container .k-actions{
          flex-direction: row-reverse;
} 
```

**Comment by nstoychev** [comment]
Fixed in v5.11.0.
